astroquery/alma/tests/test_alma_remote.py

```diff
@@ -372,11 +272,11 @@ def test_doc_example(self, temp_dir, alma):
 
         assert X30.sum() == 4  # Jul 13, 2020
         assert X31.sum() == 4  # Jul 13, 2020
-        mous1 = alma.stage_data('uid://A001/X11f/X30')
+        mous1 = alma.get_data_info('uid://A001/X11f/X30')
```

Comment:
now that the narrative documentation is tested, I think this whole test function can be removed. What do you think @keflavich?

Reply:
the `stage_data` function?  yes.

Reply:
I've commented on an awkward line, I meant to remove the whole `test_doc_example` function as hopefully of that is already covered in the narrative docs itself? (It's not verbatim as I see, e.g. the sizes are not checked, but if you think that part of the test is useful this test function may need to be renamed and scoped down?)

Reply:
oh... I don't know.  If the tests are working, is there any harm in having them duplicated in the docs and the tests?  But if it really is a duplication I have no objection to removing the copy in the test file.

Reply:
The alma tests are already among the slowest, so the only "harm" in running them multiple times is that some of the remote data tests run long, and it all adds up (this one is one of the slowest). 
Anyway, it points beyond this PR so I leave this one alone for now.